assets/xml/objects/object_crow.xml

```diff
@@ -1,24 +1,24 @@
-        <DList Name="object_crow_DL_0009A0" Offset="0x9A0" />
-        <Texture Name="object_crow_Tex_000A40" OutName="tex_000A40" Format="rgba16" Width="16" Height="16" Offset="0xA40" />
-        <Texture Name="object_crow_Tex_000C40" OutName="tex_000C40" Format="rgba16" Width="16" Height="16" Offset="0xC40" />
-        <Texture Name="object_crow_Tex_000E40" OutName="tex_000E40" Format="rgba16" Width="16" Height="16" Offset="0xE40" />
-        <Limb Name="object_crow_Standardlimb_001040" Type="Standard" Offset="0x1040" />
-        <Limb Name="object_crow_Standardlimb_00104C" Type="Standard" Offset="0x104C" />
-        <Limb Name="object_crow_Standardlimb_001058" Type="Standard" Offset="0x1058" />
-        <Limb Name="object_crow_Standardlimb_001064" Type="Standard" Offset="0x1064" />
-        <Limb Name="object_crow_Standardlimb_001070" Type="Standard" Offset="0x1070" />
-        <Limb Name="object_crow_Standardlimb_00107C" Type="Standard" Offset="0x107C" />
-        <Limb Name="object_crow_Standardlimb_001088" Type="Standard" Offset="0x1088" />
-        <Limb Name="object_crow_Standardlimb_001094" Type="Standard" Offset="0x1094" />
-        <Skeleton Name="object_crow_Skel_0010C0" Type="Flex" LimbType="Standard" Offset="0x10C0" />
+        <Animation Name="gGuayFlyAnim" Offset="0xF0" />
+        <DList Name="gGuayBodyDL" Offset="0x490" />
+        <DList Name="gGuayRightWingTipDL" Offset="0x5E0" />
+        <DList Name="gGuayRightWingBodyDL" Offset="0x6B0" />
+        <DList Name="gGuayLeftWingTipDL" Offset="0x770" />
+        <DList Name="gGuayLeftWingBodyDL" Offset="0x840" />
+        <DList Name="gGuayTailDL" Offset="0x900" />
+        <DList Name="gGuayUpperTailDL" Offset="0x9A0" />
+        <Texture Name="gGuayBodyTex" OutName="body" Format="rgba16" Width="16" Height="16" Offset="0xA40" />
+        <Texture Name="gGuayEyeTex" OutName="eye" Format="rgba16" Width="16" Height="16" Offset="0xC40" />
+        <Texture Name="gGuayTailTex" OutName="tail" Format="rgba16" Width="16" Height="16" Offset="0xE40" />
+        <Limb Name="gGuayLimbRoot" Type="Standard" EnumName="OBJECT_CROW_LIMB_ROOT" Offset="0x1040" />
+        <Limb Name="gGuayLimbBody" Type="Standard" EnumName="OBJECT_CROW_LIMB_BODY" Offset="0x104C" />
+        <Limb Name="gGuayLimbRightWingBody" Type="Standard" EnumName="OBJECT_CROW_LIMB_RIGHT_WING_BODY" Offset="0x1058" />
+        <Limb Name="gGuayLimbRightWingTip" Type="Standard" EnumName="OBJECT_CROW_LIMB_RIGHT_WING_TIP" Offset="0x1064" />
+        <Limb Name="gGuayLimbLeftWingBody" Type="Standard" EnumName="OBJECT_CROW_LIMB_LEFT_WING_BODY" Offset="0x1070" />
+        <Limb Name="gGuayLimbLeftWingTip" Type="Standard" EnumName="OBJECT_CROW_LIMB_LEFT_WING_TIP" Offset="0x107C" />
+        <Limb Name="gGuayLimbUpperTail" Type="Standard" EnumName="OBJECT_CROW_LIMB_UPPER_TAIL" Offset="0x1088" />
+        <Limb Name="gGuayLimbTail" Type="Standard" EnumName="OBJECT_CROW_LIMB_TAIL" Offset="0x1094" />
```

Comment:
Don't forget we suffix the extracted asset with its corresponding type, in this case the suffix should be `Limb`
```suggestion
        <Limb Name="gGuayRootLimb" Type="Standard" EnumName="OBJECT_CROW_LIMB_ROOT" Offset="0x1040" />
        <Limb Name="gGuayBodyLimb" Type="Standard" EnumName="OBJECT_CROW_LIMB_BODY" Offset="0x104C" />
        <Limb Name="gGuayRightWingBodyLimb" Type="Standard" EnumName="OBJECT_CROW_LIMB_RIGHT_WING_BODY" Offset="0x1058" />
        <Limb Name="gGuayRightWingTipLimb" Type="Standard" EnumName="OBJECT_CROW_LIMB_RIGHT_WING_TIP" Offset="0x1064" />
        <Limb Name="gGuayLeftWingBodyLimb" Type="Standard" EnumName="OBJECT_CROW_LIMB_LEFT_WING_BODY" Offset="0x1070" />
        <Limb Name="gGuayLeftWingTipLimb" Type="Standard" EnumName="OBJECT_CROW_LIMB_LEFT_WING_TIP" Offset="0x107C" />
        <Limb Name="gGuayUpperTailLimb" Type="Standard" EnumName="OBJECT_CROW_LIMB_UPPER_TAIL" Offset="0x1088" />
        <Limb Name="gGuayTailLimb" Type="Standard" EnumName="OBJECT_CROW_LIMB_TAIL" Offset="0x1094" />
```